composer.json

```diff
@@ -23,5 +23,8 @@
         "psr-0": {
             "Raygun4php": "src/"
         }
+    },
+    "config": {
+        "preferred-install": "dist"
```

Comment:
It looks like this config prop is supposed to accept an object: https://getcomposer.org/doc/06-config.md#preferred-install. Should this be `"preferred-install": { "*": "dist" }`?

Reply:
This should be fine, as I ran

```
$ composer config preferred-install dist
```

Reply:
Ah ok, their docs need a tweak maybe. If the intention is to speed up Travis builds by doing this, perhaps you could include this in the .travis.yml rather than the composer.json? Something like `composer install --prefer-dist` rather than adding it to the composer configuration for the package. I'd be much more comfortable merging that, since I'm not familiar with the implications of putting this change into the package's composer configuration.

Reply:
Friendly ping =)

Reply:
Setting the configuration in `composer.json` is better as this is related to `composer` and a central location - rather than using the option in multiple places. It also affects local installation when running `composer install` as a contributor to this project.

This does not affect installation of users of this package.

Reply:
Yeah fair enough. I guess it’s a project level config anyway?